modules/ROOT/pages/virtual-private-cloud.adoc

```diff
@@ -5,10 +5,10 @@ endif::[]
 
 The Virtual Private Cloud (VPC) offering allows you to create a virtual, private, and isolated network segment in the cloud to host your CloudHub workers.
 
-By connecting to your CloudHub VPC you can extend your corporate intranet whether on-prem data centers through a secured VPN tunnel, or one of your private AWS VPCs through a VPN peering, or using AWS direct connect. This allows CloudHub workers to access resources behind your corporate firewall. +
+By connecting to your CloudHub VPC you can extend your corporate intranet whether on-premises data centers through a secured VPN tunnel, or one of your private AWS VPCs through a VPN peering, or using AWS direct connect. This allows CloudHub workers to access resources behind your corporate firewall. +
 Keep in mind that you can't reuse your existing AWS VPC as a CloudHub VPC.
 
-The base VPC subscription includes 2 VPCs and one VPC can be associated with multiple environments. This allows you, for example, to have one isolated network for your production environment, and another for your QA and staging.
+The base VPC subscription includes two VPCs and one VPC can be associated with multiple environments. This allows you, for example, to have one isolated network for your production environment, and another for your QA and staging.
```

Comment:
to get rid of the "garden path" I'd change "and to something like ...two VPCs. One of the VPCs can be associated..."